lib/vtls/schannel.c

```diff
@@ -1591,18 +1941,68 @@ static CURLcode verify_certificate(struct connectdata *conn, int sockindex)
                                             SECPKG_ATTR_REMOTE_CERT_CONTEXT,
                                             &pCertContextServer);
 
-  if((status != SEC_E_OK) || (pCertContextServer == NULL)) {
-    failf(data, "schannel: Failed to read remote certificate context: %s",
+  if((status != SEC_E_OK) || !pCertContextServer) {
+    failf(data, "schannel: failed to read remote certificate context: %s",
```

Comment:
Please keep changes irrelevant for this new feature in a separate commit. As far as I can tell this is just a code-style change, right?